tests/babylonjs.test.mjs

```diff
@@ -69,19 +69,19 @@ test.describe('Prebid', () => {
     await injectIFrame(page, 'https://www.example.com', 'https://picsum.photos/300/250');
     await new Promise(res => setTimeout(res, 5000));
     const img = await page.evaluate(
-      () => window.scene.meshes[4].material.diffuseTexture.name
+      () => window.scene.meshes[4].material.diffuseTexture.url
     );
     expect(img.split('/').pop()).toBe('250');
   });
 
   test('A new ad creative is loaded after passing visibility check', async ({ page }) => {
     await injectIFrame(page, 'https://www.example.com', 'https://picsum.photos/300/250');
-    await new Promise(res => setTimeout(res, 15000));
+    await new Promise(res => setTimeout(res, 8000));
```

Comment:
Why did we halve the timeout?

Reply:
Initially I set it to 15 to match the refresh interval, but since the first simulated ad actually gets picked up in the first few seconds I realized I could lower it to 8 seconds, inject a new ad, and then check again after another 8 seconds, which would be one second after a refresh occurs.